src/modules/search/components/result/MapResults.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useState } from 'react';
+import ReactDOMServer from 'react-dom/server';
+import css from './MapResults.module.scss';
+import { Project } from '../../../../types/common';
+import { useTranslation } from 'react-i18next';
+import { MapContainer, TileLayer, Marker } from 'react-leaflet';
+import { Button, IconLocation, IconMap } from 'hds-react';
+import L from 'leaflet';
+import ProjectCard from './ProjectCard';
+
+type Props = {
+  searchResults: Project[];
+  closeMap: () => void;
+};
+
+const MAP_URL = 'https://tiles.hel.ninja/styles/hel-osm-bright/{z}/{x}/{y}.png';
+
+const MapResults = ({ searchResults, closeMap }: Props) => {
+  const { t } = useTranslation();
+  const [activeProject, setActiveProject] = useState<Project | undefined>(undefined);
+
+  const handleMarkerClick = (targetProject: Project) => {
+    setActiveProject(targetProject);
+  };
+
+  const getMarkerColor = (state?: string) => {
+    switch (state) {
+      case 'PRE_MARKETING':
+        return '#000000';
+      case 'ON_SALE':
+        return '#0000bf';
+      default:
+        return '#0000bf';
+    }
+  };
+
+  const getMarkerIcon = (project: Project) => {
+    const color = getMarkerColor(project.state_of_sale);
+    const isActive = project.uuid === activeProject?.uuid;
+
+    const element = (
+      <div
+        style={{
+          marginLeft: -14,
+          marginTop: -14,
+          width: 40,
+          height: 40,
+          backgroundColor: isActive ? color : 'white',
+          borderRadius: '50%',
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+        }}
+      >
+        <IconLocation size={'m'} color={isActive ? 'white' : color} />
+      </div>
+    );
+
+    /*if (project.building_type === 'hitas') {
```

Comment:
is this actually a comment for future implementor? and not commented code

Reply:
Yes

Reply:
Actually it is some leftover, I will remove